vaadin-combo-box-flow-demo/src/main/java/com/vaadin/flow/component/combobox/demo/ComboBoxView.java

```diff
@@ -296,6 +297,41 @@ private void lazyLoading() {
         addCard("Lazy loading with callbacks", comboBox);
     }
 
+    private void helperText() {
+        Div div = new Div();
+        // begin-source-example
+        // source-example-heading: Helper text and helper component
+        ComboBox<String> helperTextCombobox = new ComboBox<>();
+        helperTextCombobox
+              .setItems("Java", "Python", "C++", "Scala", "JavaScript");
+        helperTextCombobox
+              .setHelperText("Choose the language you are most familiar with");
+
+        ComboBox<String> helperComponentCombobox = new ComboBox<>();
+        helperComponentCombobox
+              .setItems("North America", "South America", "Africa", "Europe",
+                    "Asia", "Australia", "Antarctica");
+        helperComponentCombobox
+              .setHelperComponent(new Span("Select the biggest continent"));
+
+        ComboBox<String> helperTextAbove = new ComboBox<>();
+        helperTextAbove.setItems("Option 1", "Option 2");
+        helperTextAbove.setHelperText(
+              "Helper text positioned above the field using `helper-above-field` theme");
+        helperTextAbove.getElement().getThemeList()
+              .set("helper-above-field", true);
```

Comment:
To align with other components demo, I believe this could be moved to the "Theme Variants" tab.

Reply:
Done